### PR TITLE
11.0 mig account fiscal position partner type

### PR DESCRIPTION
**Pull request by RodrigoBM**
add cherry pick
#206

Issue: OCA/l10n-spain#1718

### CODE REVIEW

File: account_fiscal_position_partner_type/models/account_fiscal_position.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        default=lambda self: self._default_fiscal_position_type(),
+    )
+
+    @api.model
+    def _default_fiscal_position_type(self):
+        fiscal_position_type = None
+        if self.env['res.company']._company_default_get():
+            fiscal_position_type = self.env[
+                'res.company']._company_default_get(
+                    )[0].default_fiscal_position_type
```
Comment (angelmoya) [inline account_fiscal_position_partner_type/models/account_fiscal_position.py]: You can use:

self.env.user.company_id.default_fiscal_position_type

File: account_fiscal_position_partner_type/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            "selection"
+        ]
+
+    @api.model
+    def _default_fiscal_position_type(self):
+        fiscal_position_type = None
+        if self.env['res.company']._company_default_get():
+            fiscal_position_type = self.env[
+                'res.company']._company_default_get(
+                    )[0].default_fiscal_position_type
```
Comment (angelmoya) [inline account_fiscal_position_partner_type/models/res_partner.py]: Also here:

self.env.user.company_id.default_fiscal_position_type

**Review by angelmoya — COMMENTED** [review]
Code review

**Review by angelmoya — APPROVED** [review]
:+1:

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**